spec/dummy/config/environments/test.rb

```diff
@@ -13,7 +13,7 @@
   config.eager_load = false
 
   # Configure static file server for tests with Cache-Control for performance.
-  if Rails.version.starts_with?("5")
+  if Rails.version.start_with?("5")
```

Comment:
I think we might still want `Cache-Control` to be set for Rails 6 as well now, do we?

Also, I searched through the repo and I found several other deprecated `starts_with?` calls within the specs. Do you mind checking for those and updating them as well?